src/convertToObject.js

```diff
@@ -8,11 +8,21 @@
  * and values are the values of related CSS properties
 
  * @param {string} sourceString
- *
+//  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const finalArray = sourceString.split(';')
+    .map(element => (element.split(':')))
+    .filter(group => (group.length > 1));
```

Comment:
parentheses around return statement are not necessary since the statement is short